src/bloom.h

```diff
@@ -103,7 +104,11 @@ class CBloomFilter
 
 /**
  * RollingBloomFilter is a probabilistic "keep track of most recently inserted" set.
- * Construct it with the number of items to keep track of, and a false-positive rate.
+ * Construct it with the number of items to keep track of, and a false-positive
+ * rate. Unlike CBloomFilter, by default nTweak is set to a cryptographically
+ * secure random value for you. Similarly rather than clear() the method
```

Comment:
Right - 32 bit seems to be enough for this application. Trying to somehow enumerate them by probing for false positives would require a ton of probing, and be a DoS in the first place, which should be handled by other means.